### ISSUE TITLE
Create many to many relationship

### ISSUE THREAD

**Issue by NickSchimek** [codeforpdx/dwellingly-app#370]
**Purpose**

Create a many to many relationship  between Properties table and the users table

Possibly name the new table `PropertyAssignments`

**Acceptance Criteria**

- [x] users do not have to be assigned a property
- [x] appropriate Tests added

- any other rules you an encounter while building this out that makes logical sense to have in here is probably OK to add.

related issue #369 and #353 
moved other criteria to #394

**Comment by sokampdx** [comment]
@NickSchimek I am able to get the many to many relationship working with all the tests fixed. However, the constraint regarding 1,3 and 4 has not been implemented. Since I have spend a lot of time on this ticket, I like to move the other criteria into another ticket so the front end is able to use the many-to-many relationship. Let me know and I'll create the other ticket.

**Comment by NickSchimek** [comment]
Excellent, Sounds good @sokampdx , Let's get the other ticket created to cover the validations and label it with `blocked` - I think we should refactor to use marshmallow for input validation, once that is done we can unblock the ticket. Tomorrow I should create tickets to remove the reqparse and use marshmallow.  - I think that would be a good first step for all the refactorings to come.
